CHANGES.md

```diff
@@ -80,11 +80,16 @@ To be released.
      render the favicon in terminal emulators that do not support 24-bit
      colors.  [[#168], [#282] by Hyeonseo Kim]
 
+  - Fixed a bug where the getAsciiArt tests failed on terminal emulators with 
+    different color support capabilities. Enhanced color detection to support true 
+    color, 256-color, and non-color environments with appropriate test coverage. [[#304] by Hyeonseo Kim]
```

Comment:
Since we haven't released a new version yet after adding 256-color output functionality to `fedify node`, you can just include it in the previously added changeset, like:

```markdown
  -  Fixed a bug where the `fedify node` command had failed to correctly
     render the favicon in terminal emulators that do not support 24-bit
     colors.  [[#168], [#282], [#304] by Hyeonseo Kim]
```

Reply:
changed